src/Antiforgery/test/DefaultAntiforgeryTokenStoreTest.cs

```diff
@@ -237,6 +329,7 @@ public async Task GetRequestTokens_ReadFormAsyncThrowsIOException_ThrowsAntiforg
         var ioException = new IOException();
         var httpContext = new Mock<HttpContext>();
 
+        httpContext.Setup(r => r.Features).Returns(new FeatureCollection());
```

Comment:
Mock context doesn't have `Features` initialized. It is now needed to endpoint-level source configuration. Some mocked tests in other projects could probably fail for same reason as I run tests only for Antiforgery project